src/Softeq.NetKit.Chat.Data.Persistent.Sql/UnitOfWork.cs

```diff
@@ -10,6 +13,7 @@ namespace Softeq.NetKit.Chat.Data.Persistent.Sql
 {
     public class UnitOfWork : IUnitOfWork
     {
+        private const int DefaultTransactionTimeout = 1;
```

Comment:
Can we inject this parameter?

Reply:
probably it would be better to add Minutes like DefaultTransactionTimeoutMinutes

Reply:
Done